setup.py

```diff
@@ -94,10 +94,12 @@ def err(*args, **kwargs):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
         'Topic :: Software Development',
         'Topic :: Scientific/Engineering',
         'Topic :: Utilities',
     ],
+    install_requires=install_requires,
```

Comment:
I was wondering that, too. Don't know what the author's usecase is for people not installing packages in an "official" way. I use conda and pip interchangeably more or less, and have never encountered problems with package discovery so far.